src/backend/aspen/api/main.py

```diff
@@ -32,19 +32,40 @@
 
 
 def get_allowed_origins() -> List[str]:
-    allowed_origins = []
-    deployment = os.getenv("DEPLOYMENT_STAGE")
+    """Allowed origins for cross origin requests. See `CORSMiddleware`.
+
+    Gets list of all allowed origins for cross origin requests. This is pretty
+    simple right now: we only serve the FE of our app and Galago with content
+    from our BE, so that's all this needs to cover. For ease, the Galago URLs
+    are just hardcoded. If we ever need to start handling a lot more CORS
+    requests or the Galago URLs become more dynamic, we should reevaluate
+    current process.
+    """
+    GALAGO_URLS = (
+        "https://galago.czgenepi.org",
+        "https://galago-labs.czgenepi.org",
+    )
+    allowed_origins = list(GALAGO_URLS)
     frontend_url = os.getenv("FRONTEND_URL")
-
-    if deployment not in ["gestaging", "geprod", "staging", "prod"]:
-        allowed_origins.extend(
-            [r"http://.*\.genepinet\.localdev:\d+", r"^http://localhost:\d+"]
-        )
     if frontend_url:
         allowed_origins.append(frontend_url)
     return allowed_origins
 
 
+def get_allowed_origin_regex() -> Optional[str]:
+    """Allowed origin regex for cross origin requests. See `CORSMiddleware`.
+
+    This is here to enable working on Galago in localdev. Since we don't
+    have a standard port to run Galago on in localdev (everybody wants 3000),
+    this is just a catchall for all ports to enable CORS in localdev.
+    """
+    allowed_origin_regex = None  # default case, do not use regex for CORS
+    deployment = os.getenv("DEPLOYMENT_STAGE")
+    if deployment not in ["gestaging", "geprod", "staging", "prod"]:  # =>local
```

Comment:
I think this "deployment" check is kinda silly -- can we just check for the equivalent of "if deployment == 'localdev'" ?

Reply:
Sure! I was just using what was there, I don't really know that aspect very well so I didn't touch the prior code. In the docker-compose for backend, I see `DEPLOYMENT_STAGE=local`, so this should `local` for the check, right? That's the only thing I have to care about here?

Reply:
Yeah, I think that's it.